README.md

```diff
@@ -46,7 +46,10 @@ Debian/Ubuntu/Mint:    `$ sudo apt-get install build-essential libxmu-dev libxi-
 RedHat/CentOS/Fedora:  `$ sudo yum install libXmu-devel libXi-devel libGL-devel git`
 
 ##### Build
-
+	#For Unix/Mac systems (see bellow for more information)
```

Comment:
> From a downloaded tarball or zip archive

make extensions is for building from git repo, not for snapshots, and not intended for end-users.  (It may well fail!)